otx/algorithms/detection/configs/detection/resnet50_deformable-detr/data_pipeline.py

```diff
@@ -0,0 +1,115 @@
+"""Data pipeline for Deformable DETR."""
+# dataset settings
+dataset_type = "CocoDataset"
+data_root = "data/coco/"
+img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+
+# train_pipeline, NOTE the img_scale and the Pad's size_divisor is different
```

Comment:
Could you explain why it is different?

Reply:
I hope I can explain this, but I just brought data_pipeline.py from mmdetection.